prelude.links

```diff
@@ -1226,7 +1227,7 @@ fun forkSync(f) {
   var _ = spawn {
     var c = accept(ap);
     var c = f(c);
-    close(c)
+    closeBang(c)
```

Comment:
Maybe we should amend the lexical definition of variables to include symbols such as exclamation marks.

Reply:
I did not mean to request a change for the above, rather it was more of a general comment.